app/templates/buyers/brief_publish_confirmation.html

```diff
@@ -35,9 +35,8 @@
         {% include "toolkit/page-heading.html" %}
     {% endwith %}
     <p class="padding-bottom-small">All requirements are published on the Digital Marketplace where anyone can see them.</p>
-
-    <p><strong>Suppliers will be able to apply for:</strong></p>
-    <p class="padding-bottom-small">2 weeks ending at midnight.</p>
+    <p class="padding-bottom-small">All requirements are open for {{ dates.application_open_weeks }} weeks.</p>
+    <p class="padding-bottom-small">If you publish your requirements today ({{ dates.today }}), suppliers will be able to apply until {{ dates.closing_time }}, {{ dates.closing_date }}.</p>
```

Comment:
It's the time localisation again. Since we removed it for dates, we can run the date through the display filter but we can't do the same for time since it'll be transformed into 00:59. And the same will happen if we run it through any of the date and time filters. That's why we decided to display the time like this for now.